en/flight_modes_rover/ackermann.md

```diff
@@ -18,7 +18,7 @@ Manual modes require stick inputs from the user to drive the vehicle.
 
 The sticks provide the same "high level" control effects over direction and rate of movement in all manual modes:
 
-- `Left stick up/down`: Drive the rover forwards/backwards (controlling forward speed)
+- `Left stick up/down`: Drive the rover forwards/backwards (controlling longitudinal speed)
```

Comment:
I guess longitudinal is OK. I preferred the original. I.e. this means "controlling forward backward speed".

Why? Well when someone says longitudinal I'm thinking, now what does that mean?

```suggestion
- `Left stick up/down`: Drive the rover forwards/backwards (controlling speed)
```